Protecs/src/test/java/uk/ac/ucl/protecs/objects/diseases/CoronavirusInfectiousBehaviourTesting.java

```diff
@@ -0,0 +1,308 @@
+package uk.ac.ucl.protecs.objects.diseases;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import uk.ac.ucl.protecs.objects.Person;
+import uk.ac.ucl.protecs.sim.Params;
+import uk.ac.ucl.protecs.sim.WorldBankCovid19Sim;
+import uk.ac.ucl.swise.behaviours.BehaviourNode;
+import uk.ac.ucl.protecs.helperFunctions.*;
+
+public class CoronavirusInfectiousBehaviourTesting {
+	// ==================================== Testing ==================================================================
+	// === These tests are designed to ensure that the transition between different infectious behaviour nodes are ===
+	// === happening as they should do. Each of the behaviour nodes are forced into the population and then the ======
+	// === infectious behaviour nodes that are meant to be transitioned to via the model's inner workings are checked =
+	// === against the infectious behaviour nodes that were activated by the simulation. ==============================
+	
+	@Test
+	public void TestSusceptibleEndpoints() {
+		// create a simulation and start
+		WorldBankCovid19Sim sim = helperFunctions.CreateDummySim("src/test/resources/InfectiousBehaviourTestParams.txt", false);
```

Comment:
missing folder?

Reply:
test/resources would not get it's stupid self into github, tried several things. Just going to house the tests to run from main/resources now!